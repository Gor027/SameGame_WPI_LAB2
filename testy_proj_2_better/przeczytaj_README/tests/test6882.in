g  gg## ggggg gg 
g# #  ggg#g # g 
g gg#g g#g#g# #
g### #g g # g ##g#ggg # g #gggg  g g #g####g #ggg #    #g# #g
##g g#  g #gg # #gg ##  ##g g# g g g g# g #g g
  g   g # g### ### gg# #g g#gg###  #g #gg##gg#g #gggg# #g# # ###gg##g#g#gg
  #  g###g##### ##   g#g##  gggg##gg g g  g   g  # # g g  g## gg 
###g #ggg#### gg# ##g#  #  g ##
