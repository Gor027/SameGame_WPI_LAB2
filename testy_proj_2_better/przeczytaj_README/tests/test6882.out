       g g g
       g g # g
    ## # #g# g
    gg gg#gg # g       g    g
ggg #ggg##gg #g##  g#  #   g#g    g      #
g## #gg#gg## ###ggg##  g #ggg#g g g #g## #g #g       #   #
ggggg###g########gggg####g#ggg##g # g#gg##g gg gg#g ##g# #g ##
gggg#ggg#####gg####g##g##ggg##g##gggg#gg#ggg#gg##g#gggg#g###gggg##g#g#gg
